apps/server/src/modules/common-cartridge/service/common-cartridge-export.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import { FileRecord } from '@src/modules/files-storage/entity';
+import { FilesStorageService } from '@src/modules/files-storage/service';
+
+@Injectable()
+export class CommonCartridgeExportService {
+	constructor(private readonly filesService: FilesStorageService) {}
+
+	public async findCourseFileRecords(courseId: string): Promise<FileRecord[]> {

```

Comment:
Do NOT use an Entity from another domains as a return here. Create an approriate DTO